[incubator/crate] Add CrateDB (#16318)
helm/charts: incubator/crate/NOTES.txt

If you want to get a sneak peek at the CrateDB UI, you can forward the `4200` port.

Assuming deployment is ready...

```
kubectl port-forward crate-0 4200:4200
```

> Now open an explorer at http://localhost:4200, you should see the CrateDB's Admin UI.

Reviewer: ```suggestion
Now open a browser at http://localhost:4200, you should see the CrateDB's Admin UI.
```
